### PR TITLE
Modules

### LINKED ISSUE #56: feat: Support for modules
Turn off different sections of commands.

### PR DESCRIPTION
**Pull request by PandaGamersHDNL**
modules for misc, notes, reminders, courses, custom commands and wiki

### CODE REVIEW

**Comment by DrunkenToast** [comment]
A potential alternative would be to add a catagory to a command and then checking if these catagories are disabled rather than putting a check in each command.

**Review by DrunkenToast — COMMENTED** [review]
A potential alternative would be to add a catagory to a command and then checking if these catagories are disabled rather than putting a check in each command.

**Comment by PandaGamersHDNL** [comment]
use embed maker for error message

**Comment by PandaGamersHDNL** [comment]
ready: don't know what else i would add

File: src/discord.ts [hunk 1]
```diff
@@ -123,10 +124,14 @@ export async function buildClient(shard: number, shardCount: number): Promise<Cl
       return;
     }
 
-    for (const command of commands.concat(guildConfig.commands)) {
+    for (const command of commands.concat(guildConfig.modules['customCommands'] === false ? [] : guildConfig.commands)) {
```
Comment (xunafay) [inline src/discord.ts]: concat logic outside of for loop to improve readability

**Review by xunafay — APPROVED** [review]

**Review by DrunkenToast — APPROVED** [review]

**Pull request merged**